src/tests/ftest/osa/osa_offline_drain.py

```diff
@@ -61,20 +65,27 @@ def run_offline_drain_test(self, num_pool, data=False):
             pool[val].nvme_size.value = int(pool[val].nvme_size.value /
                                             num_pool)
             pool[val].create()
-            pool_uuid.append(pool[val].uuid)
             self.pool = pool[val]
+            if drain_during_aggregation is True:
+                test_seq = self.ior_test_sequence[1]
+                self.pool.set_property("reclaim", "disabled")
+            else:
+                test_seq = self.ior_test_sequence[0]
+
```

Comment:
Any specific reason we need different test sequences for aggregation enabled vs disabled?
I see the change is mainly of the transfer size. Can you help me understand why we need to do this.

Reply:
I thought we need less than 4K transfer size data (or delete data) for aggregation to happen. The other test data I am not sure how I can do write and delete... With less than 4K data, I think we can be confident that aggregation is going to occur? Am I wrong?

Reply:
Yeah as you said there are two ways for aggregation to happen, 4K transfer size data or delete data. We can be confident if aggregation is going to be happening by deleting the data. And you don't need to change your test sequence for that either. It will simply your test and may be you wouldn't even need to do the looping of exclude/reintegrate for 100 secs as described below then. All you will need to do is, create pool, disable aggregation, write data, enable aggregation, delete some data or all as needed, exclude, reintegrate, verify aggregation was successful.

Reply:
Done.